Testscripts/Windows/FCOPY-NEGATIVE.ps1

```diff
@@ -124,90 +124,104 @@ function Main {
     #
     Write-LogInfo "Info: Step 1: fcopy file to vm when target folder is immutable"
 
-    # Verifying if /tmp folder on guest exists; if not, it will be created
-    .\Tools\plink.exe -C -pw $VMPassword -P $VMPort $VMUserName@$ipv4 "sudo [ -d /test ] || mkdir /test ; chattr +i /test"
-
+    # Verifying if /test folder on guest exists; if not, it will be created
+    $cmd = "[ -d /test ] || mkdir /test ; chattr +i /test"
+    Run-LinuxCmd -username $VMUserName -password $VMPassword -ip $Ipv4 -port $VMPort $cmd -runAsSudo
     if (-not $?) {
         Write-LogErr "Fail to change the permission for /test"
```

Comment:
I am asking if the line 131 and 132 are missing "return" statement.

Reply:
Add the 'return' statement.